easybuild/easyconfigs/p/Paraview/Paraview-4.3.1-intel-2015a-Python-2.7.10-mpi.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'CMakeMake'
+
+name = 'Paraview'
+version = '4.3.1'
+
+homepage = "http://www.paraview.org"
+description = "Paraview is a scientific parallel visualizer."
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % download_suffix]
+sources = ["ParaView-v%(version)s-source.tar.gz"]
+
+patches = ['missingheader-%(version)s.patch']
+
+python = 'Python'
+pyver = '2.7.10'
+versionsuffix = '-%s-%s-mpi' % (python, pyver)
```

Comment:
problem is, if you really mean it to be consistent with all, it is needed also to add build modes/flags, opt options etc; we merely end up with a hash function of some build configuration. See, #1689 for another variation of this issue/need. The faster we agree that this is simply a hash of the build setup, the sooner we'll come to agreement about what it should look like.